plugins/network/NetClass.hpp

```diff
@@ -2,37 +2,110 @@
 #define __NETCLASS_HPP__
 
 #include <plugin.hpp>
+#include <memory>
```

Comment:
plugin.hpp includes windows.h, it's probably better to include it after std stuff just in case (macro conflicts etc.)